bin/study.js

```diff
@@ -1,5 +1,18 @@
 // Repeat Ruby Lab for JS
 // let myArray = [1, 2, 3, 4, 5];
+'use strict'
+
+let arr = [];
+arr.length = 5;
+arr[arr.length+1];
+arr[-3];
+arr[9];
+arr.splice(-5, 3, 12, -49);
+arr.forEach(function(e,i,a){
+  if(e>0){
+    console.log(a[e]);
+  }
```

Comment:
`.forEach` could certainly work, but it may be overkill, since all we want is to `console.log` the items in the array. You could more easily so `console.log(e);` - does that make sense?